packages/eas-cli/src/utils/statuspageService.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  StatuspageServiceFragment,
+  StatuspageServiceName,
+  StatuspageServiceStatus,
+} from '../graphql/generated';
+import { StatuspageServiceQuery } from '../graphql/queries/StatuspageServiceQuery';
+import Log from '../log';
+
+export async function maybeWarnAboutEasOutagesAsync(
+  serviceNames: StatuspageServiceName[]
+): Promise<void> {
+  const services = await getStatuspageServiceAsync(serviceNames);
+
+  for (const service of services) {
+    warnAboutServiceOutage(service);
+  }
+}
+
+function warnAboutServiceOutage(service: StatuspageServiceFragment): void {
+  const humanReadableServiceName: Record<StatuspageServiceName, string> = {
+    [StatuspageServiceName.EasBuild]: 'EAS Build',
+    [StatuspageServiceName.EasSubmit]: 'EAS Submit',
+    [StatuspageServiceName.EasUpdate]: 'EAS Update',
+  };
+
+  if (service.status !== StatuspageServiceStatus.Operational) {
+    Log.warn(
+      `⚠️  ${
```

Comment:
it's fine to inline icons this way, but try using the `figures` package, it might provide better fallbacks on systems where icons are not available.

Reply:
<img width="718" alt="Screenshot 2022-08-25 at 13 37 50" src="https://user-images.githubusercontent.com/55145344/186654947-57390e1c-c46a-4dda-a011-ff34e9c25b8f.png">
This is how it looks like after the update

Reply:
the previous icon was definitely looking better on macos, on my system previous version and the current one look the same. I'm not sure what would happen if icon fonts are missing.

I do not have any strong opinion on which approach to take.

Reply:
I think that the one using the `figures` package is definitely safer because it has a fallback which is `!!`, so I would say that we can go with it, although maybe @brentvatne has a different opinion 🤔

Reply:
i think we can leave out the icon and just show the text :) the color is a strong enough attention draw